retrofit/src/main/java/retrofit/RestAdapter.java

```diff
@@ -415,8 +415,8 @@ public Builder setConverter(Converter converter) {
     }
 
     /**
-     * The error handler allows you to customize the type of exception thrown for errors on
-     * synchronous requests.
+     * The error handler allows you to customize
+     * {@link retrofit.client.Response response} exceptions.
```

Comment:
Do you have preferred verbiage? I'd rather fix it in all places and have it be completely clear.